charts/datahub/values-prod.yaml

```diff
@@ -0,0 +1,543 @@
+# Values to start up datahub after starting up the datahub-prerequisites chart with "prerequisites" release name
+# This chart has been copied from upstream and the configuration changed as needed.
+datahub-gms:
+  enabled: true
+  image:
+    repository: acryldata/datahub-gms
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 2Gi
+    requests:
+      cpu: 100m
+      memory: 1Gi
+  livenessProbe:
+    initialDelaySeconds: 60
+    periodSeconds: 30
+    failureThreshold: 8
+  readinessProbe:
+    initialDelaySeconds: 120
+    periodSeconds: 30
+    failureThreshold: 8
+  podAnnotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "4318"
+ 
+datahub-frontend:
+  enabled: true
+  image:
+    repository: acryldata/datahub-frontend-react
+    tag: "v0.13.2"
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 1400Mi
+    requests:
+      cpu: 100m
+      memory: 512Mi
+  ingress:
+    enabled: false
+  defaultUserCredentials: {}
+  service:
+    extraLabels: {}
+  podAnnotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "4318"
+  extraEnvs:
+    - name: AUTH_OIDC_ENABLED
+      value: "true"
+    - name: AUTH_OIDC_DISCOVERY_URI
+      value: https://accounts.google.com/.well-known/openid-configuration
+    - name: AUTH_OIDC_BASE_URL
+      value: "https://datahub.canonical.com"
+    - name: AUTH_OIDC_SCOPE
+      value: "openid profile email"
+    - name: AUTH_OIDC_CLIENT_ID
+      valueFrom:
+        secretKeyRef:
+          name: datahub-oidc-secret
+          key: id
+    - name: AUTH_OIDC_CLIENT_SECRET
+      valueFrom:
+        secretKeyRef:
+          name: datahub-oidc-secret
+          key: secret
+    - name: AUTH_OIDC_USER_NAME_CLAIM
+      value: email
+    - name: HTTP_PROXY_HOST
+      value: squid.internal
+    - name: HTTP_PROXY_PORT
+      value: "3128"
+    - name: HTTPS_PROXY_HOST
+      value: squid.internal
+    - name: HTTPS_PROXY_PORT
+      value: "3128"
+    - name: HTTP_NON_PROXY_HOSTS
+      value: "localhost|datahub-datahub-gms"
+    - name: AUTH_JAAS_ENABLED
+      value: "false"
+    - name: AUTH_VERBOSE_LOGGING
+      value: "true"
+
+acryl-datahub-actions:
+  enabled: true
+  image:
+    repository: acryldata/datahub-actions
+    tag: "v0.0.15"
+  resources:
+    limits:
+      cpu: 500m
+      memory: 2Gi
+    requests:
+      cpu: 300m
+      memory: 256Mi
+        # extraEnvs:
```

Comment:
question: Why no proxy here?

Reply:
Seems like they were not needed. Very likely they were added in the attempt to solve some issue, and then removed afterwards. I will delete the commented part/